src/seedu/addressbook/data/person/TimeTable.java

```diff
@@ -0,0 +1,49 @@
+package seedu.addressbook.data.person;
+
+public class TimeTable {
+
+    private TimeSlot[][] timeslot;
+    private boolean isPrivate;
+    private boolean isPublic;
+    private boolean isFinal;
+    private boolean isNotFinal;
+
+    public TimeTable(){
+
+    }
+
+    public TimeSlot getSlot(TimeSlot[][] slots,String date,String period){
+        int row = 0;
+        int column =0;
+
+        switch(date){
```

Comment:
minor comment - you could ensure that the spaces between the `=` sign in your cases are consistent.